xija/get_model_spec.py

```diff
@@ -92,16 +92,20 @@ def get_xija_model_spec(model_name, version=None, repo_path=REPO_PATH,
 
     Returns
     -------
-    dict, str
-        Xija model specification dict, chandra_models version
+    tuple of dict, str, str
+        Xija model specification dict, chandra_models version, path to model spec
+        file used
```

Comment:
might as well update the example in the doc string to show this new return value (or add a second example.